src/config/dynamic-theme-fixes.config

```diff
@@ -7273,6 +7273,11 @@ INVERT
 .page-logo
 .page-header-logo
 
+CSS
+.page-header-logo > a > svg > g {
+    --darkreader-inline-fill: ${white} !important;
```

Comment:
![image](https://user-images.githubusercontent.com/25481501/132105318-ead57659-919a-429f-af0a-7b789bf8526f.png)

It is already dark?

Reply:
> ![image](https://user-images.githubusercontent.com/25481501/132105318-ead57659-919a-429f-af0a-7b789bf8526f.png)
> 
> It is already dark?

Was trying to fix the issue of scrolling down the home page causing the logo to go dark. I found a cleaner way to fix it and modified this pull request.